Tests/conf.json

```diff
@@ -1281,6 +1281,11 @@
         {
             "playbookID": "TestHttpPlaybook"
         },
+        {
+            "integrations": "SplunkPy",
+            "playbookID": "SplunkPy parse-raw - Test",
```

Comment:
why isn't it part of the current test playbook? why do you need a new one?

Reply:
I did not want to mess up there